coderadar-test/src/test/java/io/reflectoring/coderadar/rest/useradministration/teams/ListTeamsForUserControllerIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package io.reflectoring.coderadar.rest.useradministration.teams;
+
+import static io.reflectoring.coderadar.rest.JsonHelper.fromJson;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+
+import io.reflectoring.coderadar.graph.useradministration.domain.TeamEntity;
+import io.reflectoring.coderadar.graph.useradministration.domain.UserEntity;
+import io.reflectoring.coderadar.graph.useradministration.repository.TeamRepository;
+import io.reflectoring.coderadar.graph.useradministration.repository.UserRepository;
+import io.reflectoring.coderadar.rest.ControllerTestTemplate;
+import io.reflectoring.coderadar.rest.domain.GetTeamResponse;
+import java.util.Collections;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+class ListTeamsForUserControllerIntegrationTest extends ControllerTestTemplate {
+
+  @Autowired private TeamRepository teamRepository;
+
+  @Autowired private UserRepository userRepository;
+
+  private TeamEntity teamEntity;
+  private UserEntity testUser;
+
+  @BeforeEach
+  void setUp() {
+    userRepository.deleteAll();
```

Comment:
Zu Beginn eines jeden Tests sollte die Datenbank leer sein, Aufräumarbeiten sollten entsprechend in der `@AfterEach`-Methode geschehen.

Reply:
In IntegrationTestTemplate ausgelagert.